src/protocols/index.js

```diff
@@ -1,6 +1,7 @@
 const immutable = require('immutable')
 const EthereumTypeGenerator = require('./ethereum/type-generator')
 const EthereumTemplateCodeGen = require('./ethereum/codegen/template')
+const NearTemplateCodeGen = require('./near/codegen/template')
```

Comment:
This change is making the CI fail, there's no such file.

Reply:
Spurious change that slipped through and is actually part of another PR #824, will remove that.